benchmarks/cuda/benchmark_cuda.cpp

```diff
@@ -6,13 +6,17 @@
  */
 
 // VecMem include(s).
+#include <vecmem/memory/binary_page_memory_resource.hpp>
```

Comment:
It's a very petty thing, but now that the benchmark directly depends on `vecmem::core`, please list that library explicitly in:

https://github.com/acts-project/vecmem/blob/main/benchmarks/cuda/CMakeLists.txt#L19

Of course in the current setup, since `vecmem::cuda` publicly depends on `vecmem::core`, the build succeeds just fine. This is just one of the things that I'm sensitive to with my build system hat on. :stuck_out_tongue:

Reply:
Fixed, thanks. :slightly_smiling_face: